pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -389,15 +389,30 @@ func (h *NetworkUtilsHandler) StartDHCP(nic *VIF, serverAddr net.IP, bridgeInter
 
 func (h *NetworkUtilsHandler) CreateNDPConnection(bridgeInterfaceName string, launcherPID int) error {
 	log.Log.Infof("Starting RA daemon on network Nic: %s", bridgeInterfaceName)
+
+	if isSELinuxEnabled() {
+		if err := setVirtHandlerSELinuxSocketContext(launcherPID); err != nil {
+			return fmt.Errorf("error setting next socket's selinux context: %v", err)
+		}
+
+		defer func() {
+			_ = resetVirtHandlerSELinuxSocketContext()
+		}()
+	}
+
 	ndpConnection, err := ndp.NewNDPConnection(bridgeInterfaceName)
 	if err != nil {
 		return fmt.Errorf("failed to create the RouterAdvertisement daemon: %v", err)
 	}
 
-	return ndpConnection.Export(
-		GetNDPConnectionUnixSocketPath(
-			fmt.Sprintf("%d", launcherPID),
-			bridgeInterfaceName))
+	go func() {
+		_ = ndpConnection.Export(
```

Comment:
Don't we want to at least print a log if the the `Export` fails?

Reply:
Done in https://github.com/kubevirt/kubevirt/pull/4581/commits/442efb74d0451d1ceb564b93f224eb1951dbab22